### ISSUE TITLE
Agenda/Notes for SG meeting 2015-06-11

### ISSUE THREAD

**Issue by danbri** [schemaorg/schemaorg#588]
Previous meeting: #587 
Next meeting: #775 

mail: https://groups.google.com/forum/#!topic/schema-org-sg/0xk2910ii58
# Agenda

## Discussion topics

* auto/bib extension status: Dan met f2f with Richard Wallis (bib) and Martin Hepp (auto) last week. We won't get this done for June 15 but late June should be feasible.  For bib: we should consider whether we can live with adding an Agent type above Organization/Person, see #431 and https://github.com/schemaorg/schemaorg/pull/461/files#diff-c50396bf2a61d4cf44f88edfcd4c436dR29 for details.
* AOB?

## Recent change review

Agreed from last meeting and merged into next release branch sdo-ganymede:

* Added enumerated values to payment status #518 implementation from #523 has been merged in
 * see issue for discussion of PaymentDue name, and possible refinements.
* Issue #137 (pull #514) - replacing interactionCount with InteractionCounter - awaiting @chaals review but tentatively merged in.
* Issue #526 (Improved schema for social media, pull #549) has been merged in.

Review build: http://sdo-ganymede.appspot.com/docs/releases.html#sdo-ganymede 

See test site for all above changes, plus additional fixes from community group discussions:

* Fix to #577: Amended videoFormat to indicate that it is expected on BroadcastEvent and ScreeningEvent, rather than TelevisionStation.
* Fix to #535: Broadened domain of parentOrganization to allow any Organization, rather than only LocalBusiness. Noted parentOrganization and subOrganization as inverses.
* Fix to #531: Added the lyrics property to MusicComposition.

# Meeting Notes

present: @shankarnat @vholland @scor @ajax-als @danbri 
regrets: @guha @chaals @pmika (1st 1/2 hour but we only needed 25m) @mfhepp 

RESOLVED:

 * 1. alumni / alumniOf https://github.com/schemaorg/schemaorg/issues/432 - we have not consensus to move from our narrow academic/learning sense of alumni so we will not broaden the definition at all currently.
 * 2. dataset terms: https://github.com/schemaorg/schemaorg/issues/583 - we agreed with the direction outlined in the issue by danbri.
 * 3. Nobody had any concerns with the implementation of last meeting's 3 issues; dan/vicki will figure out the Payment term name in github, please join discussion there. Practical issue of whether a single enumerated value term can be in two enumerations simultaneously  (ok conceptually but site UI might not work yet).
 * 4. approved the 3 community fixes introduced since last meeting, i.e. (see #588 notes and test sites): #577 #535 #531 

From @pmika on reading the meeting summary: "I've gone through this, thanks! I agree to all the decisions, you have our support."

**Comment by danbri** [comment]
Also can we decide what sense of alumni we intend?https://github.com/schemaorg/schemaorg/issues/432 /cc @shankarnat

**Comment by danbri** [comment]
Dataset tweak: https://github.com/schemaorg/schemaorg/issues/583

**Comment by chaals** [comment]
regrets for the meeting.

I prefer the broader sense of alumni. I'll chase up review of #137 but I may not get there before the meeting - it is actually pretty hard to understand from the paper trails what happened and why ;(

#577 #535 and #531 look OK to me

**Comment by danbri** [comment]
@vholland can you help w/ summarising #137, I also found the trail a bit tangled

**Comment by vholland** [comment]
Prior to schema.org 2.0, CreativeWork had the property "interactionCount"
for describing the number of likes, views, etc a Work received. For
example, a video may have been viewed 128 times.

The expected type was Text and the description suggested authors format the
string as something like "5:UserLikes, 10:UserComments". The string values
were subtypes of UserInteraction.

This was all messy, hard to parse, and conflicted with properties like
"commentCount".

The proposed change is:

  - Deprecate "interactionCount" in favor of "interactionStatistic", which
expects an InteractionCounter.
  - InteractionCounter has the following properties:
      - interactionService: The website or software application where the
interactions took place (e.g. Facebook, Twitter, etc.)
      - interactionType: The http://schema.org/Action representing the type
of interaction (e.g. http://schema.org/Like)
      - userInteractionCount: The number of interactions of the type on the
service


I can give example markup if that is easier.


Vicki


Vicki Tardif Holland | Ontologist | vtardif@google.com

**Comment by danbri** [comment]
Thanks @vholland. Effectively this wraps up the cleanup initially proposed over 3 years ago in https://lists.w3.org/Archives/Public/public-vocabs/2012Apr/0030.html 

There is one final implementation piece which is finishing off #465 i.e. updating the site codebase (and schema files) to allow types as well as properties to be marked as superseded and/or deprecated. Currently we have no UI (beyond textual definitions) to indicate when a term has been retracted from mainstream use.
